docs/features/kubernetes/configuration.md

```diff
@@ -48,6 +52,45 @@ kubernetes:
       exposeDashboard: true
 ```
 
+### `frontend` (optional)
+
+This is an array used to configure some frontend features.
+
+Valid values are:
+
+- `podDelete`
+
+#### `podDelete` (optional)
+
+This configures the behavior of the delete pod button in the container panel.
+
+Valid configurations are:
+
+- `enabled`
+- `restartButtonText`
+
+##### `enabled`
+
+This configuration controls the visibility of this feature.
+
+Valid values are:
+
+- `true`
+- `false`
+
+The default value is `false`.
+
+##### `restartButtonText`
```

Comment:
Sorry but this is a bit of a step in the wrong direction as it reduced the configurability compared to before. Feedback remains the same though, the translation API needs to be used for text customization.

Reply:
OK I'll revert it and implement i18n

Reply:
done :) please give it an eye.

Reply:
hey @Rugvip can I get another review please 🙏